scripts/ipfs/ipfs-swarm-cjdns.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Wait for ipfs to initalize
+attempts=15
+until [[ $( curl http://localhost:5001/api/v0/id -q 2>/dev/null) || ${attempts} -eq 0 ]]; do
+    sleep 1
+    attempts=$((attempts-1))
+done
+
+while read -r peer; do
+
+    peer=$(sudo /opt/cjdns/publictoip6 $peer)
+
+    # See if they have IPFS enabled
+    res=$(curl http://[${peer}]/nodeinfo.json)
+    if [ ! -x "$res" ]; then
+            
+        id=$(echo ${res} | jq -r -M '.services.ipfs.ID')
```

Comment:
There really should be a shared module that handles schema version checking and parsing, instead of having each module `jq` parse this. In the current model, it means once the schema updates, every module will need to update its `nodeinfo` parsing.

Reply:
comment added to #173 
Leaving it for now